bitnami/matomo/templates/_helpers.tpl

```diff
@@ -139,7 +139,7 @@ Return the database password key
 {{- if .Values.mariadb.enabled -}}
 mariadb-password
 {{- else -}}
-db-password
+    {{- printf "%s" .Values.externalDatabase.existingSecretPasswordKey -}}
```

Comment:
To maintain consistency, I think it's best to leave the default password in the helper too. [Eg](https://github.com/bitnami/charts/blob/main/bitnami/jupyterhub/templates/_helpers.tpl#L285)